xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,304 @@ migrate</screen>
     the selected nodes.
    </para>
   </sect2>
+  <sect2 xml:id="sec.depl.maintenance.upgrade.troubleshooting">
+   <title>Common Upgrade Problems and Possible Solutions</title>
```

Comment:
"possible" ?

Reply:
I would recommend this is retitled to "Troubleshooting upgrades"

Reply:
Good call. How about _Troubleshooting Upgrade Issues_? (You don't troubleshoot upgrades, no?)

Reply:
Good point - suggestion approved! :D